registry/src/main.rs

```diff
@@ -63,16 +63,21 @@ fn main() {
                         }
                     }
                 },
-                args::ConfigSubCommand::Set{input} => {
+                args::ConfigSubCommand::Set{input, what_if} => {
                     let reg_helper = match RegistryHelper::new(&input) {
```

Comment:
Did you consider making `what_if` a member of `RegistryHelper` instead of passing it to the methods?